src/views/dataaccesscoe/community-support.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="usa-layout-docs__main desktop:grid-col-9 usa-prose">
+    <h1>Community Support</h1> <!--comment!-->
+    <p>
+      The ITS JPO has created a community called the Data Access Center of Excellence (CoE) to promote best practices around data management in order to maximize utility of ITS JPO-sponsored project data.
+    </p>
+    <p>
+      The Data Access CoE aims to achieve this by executing the following objectives:
+    </p>
+    <ul>
+      <li>
+        Promoting data access and use across the ITS research portfolio
+      </li>
+      <li>
+        Piloting data management best practices, standards, templates, and data storage & discovery capabilities within ITS JPO that can be scaled Department-wide
```

Comment:
`department` should be lower case, and add a period at the end maybe.

Reply:
Agree and fixed